tests/scripts/ci.py

```diff
@@ -237,9 +277,9 @@ def docs(
         check_gpu()
 
     scripts = extra_setup + [
-        config,
-        f"./tests/scripts/task_build.sh build -j{NPROC}",
-        "./tests/scripts/task_ci_setup.sh",
+        config + f" {build_dir}",
+        f"./tests/scripts/task_build.py --build-dir {build_dir}",
+        "python3 -m pip install --user tlcpack-sphinx-addon==0.2.1 synr==0.6.0",
```

Comment:
is there a comment elsewhere like `# keep in sync with tests/scripts/ci.py` and vice verse?

Reply:
hopefully these will be gone soon after we merge #10257